cog/daily_charge.py

```
@@ -23,95 +26,242 @@
-            # pylint: disable-next = line-too-long
-            print(
-                f"{interaction.user.id},{interaction.user} Get 5 point by daily_charge {datetime.now()}"
-            )
-        end(connection, cursor)
+            return
+
+        with mysql_connection() as cursor:  # SQL 會話
+            user = interaction.user
+
+            # get now time and combo
+            now = datetime.now().replace(microsecond=0)
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)